config/routes.rb

```diff
@@ -6,11 +6,12 @@
 Rails.application.routes.draw do
 
   constraints PortConstraint.new(PrometheusExporterPort.call) do
-    require 'sidekiq/prometheus/exporter'
     require 'yabeda/prometheus/mmap'
 
-    mount Sidekiq::Prometheus::Exporter, at: '/metrics'
-    mount Yabeda::Prometheus::Exporter, at: '/yabeda-metrics'
+    # TODO: enable this /metrics endpoint when we're happy with the current metrics in SaaS
+    # mount Yabeda::Prometheus::Exporter, at: '/metrics', as: :metrics
+    # DEPRECATED: this endpoint will be removed in future versions
+    mount Yabeda::Prometheus::Exporter, at: '/yabeda-metrics', as: :deprecated_metrics
```

Comment:
What is the purpose of the `as` parameter here?

Reply:
When the same app is mounted at the same path, this is required. So, when line `:12` is uncommented, it will still be OK with the `as:`.

Reply:
got it, in preparation for the other